bindings/ios/MEGABanner.h

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file MEGAUser.h
+ * @brief Represents an user in MEGA
+ *
+ * (c) 2013-2014 by Mega Limited, Auckland, New Zealand
+ *
+ * This file is part of the MEGA SDK - Client Access Engine.
+ *
+ * Applications using the MEGA API must present a valid application key
+ * and comply with the the rules set forth in the Terms of Service.
+ *
+ * The MEGA SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * @copyright Simplified (2-clause) BSD License.
+ *
+ * You should have received a copy of the license along with this
+ * program.
+ */
+
```

Comment:
add `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`

Reply:
It's hard to tell whether the SDK return values are nullable or not, to put `assume_nonnull` is dangerous as putting a `!`

Reply:
But we still need to audit all OC code to make it friendly to swift - https://confluence.developers.mega.co.nz/display/IOS/Swift+Style+Guide#SwiftStyleGuide-Objective-C

Reply:
Yep, I mean, if we add `non_null` to a, for example, function, but SDK returns `null` value, it leads to problems. I had similarly issue for `avatar` sdk apis.

Reply:
we can make them all nullable

Reply:
got it

Reply:
thanks! at least we make it explicit, rather than having the opportunity to use `!`